src/main/java/com/mindee/parsing/receipt/ReceiptV4Inference.java

```diff
@@ -0,0 +1,48 @@
+package com.mindee.parsing.receipt;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.mindee.parsing.EndpointInfo;
+import com.mindee.parsing.SummaryHelper;
+import com.mindee.parsing.common.Inference;
+import com.mindee.parsing.common.Page;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+/**
+ * The receipt V4 inference.
+ */
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+@EndpointInfo(endpointName = "expense_receipts", version = "4")
+public class ReceiptV4Inference extends Inference {
+
+  /**
+   * The prediction on each pages of the document.
+   */
+  @JsonProperty("pages")
+  private ArrayList<Page<ReceiptV4DocumentPrediction>> pagesPrediction;
+  /**
+   * The prediction on the document level.
+   */
+  @JsonProperty("prediction")
+  private ReceiptV4DocumentPrediction documentPrediction;
+
+  @Override
+  public String toString() {
```

Comment:
This should be done in a way where we can change the output format in a single place, rather than having to change in each class.

You could maybe implement this in the parent class, or define a helper function where you just pass the document and page predicitons.

Reply:
Yes I will in Passport V1 !
I was waiting to have duplicate code before trying to refacto.

Reply:
:+1: